lib/meaning-cloud.rb

```diff
@@ -17,10 +17,14 @@ def self.configure
 
   # Main configuration class.
   class Configuration
-    attr_accessor :key
+    attr_accessor :key,
+      :language,
+      :topic_types
 
     def initialize
       @key = nil
+      @language = :en
+      @topic_types = 'ec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.